docs/tutorials/query-language.md

```diff
@@ -116,6 +116,66 @@ Example:
 let teens = realm.objects('Contact').filtered('SUBQUERY(friends, $friend, $friend.age > 21 AND $friend.city = "SF").@count > 0');
 ```
 
+### Queries on lists of primitives
+
+The query syntax for a list of primitive values is mostly the same as querying a list of objects via links or lists.
+There are however, some small differences. To illustrate, let's construct a movie database, where each movie has an anyonomous star rating, and a list of string tags.
+Note that a normal list of objects should be used to model more complex relationships, such as if the start or tags should belong to a specific user.
```

Comment:
What do you mean with "start ... belong to a user"? 
```suggestion
Note that a normal list of objects should be used to model more complex relationships, such as if the tags should belong to a specific user.
```

Reply:
That can probably be left out.

Reply:
ok I'll remove this